CHANGELOG.md

```diff
@@ -4,18 +4,35 @@ Changelogs for versions not listed here can be found at https://github.com/DataD
 
 ---
 
-## 2.9.6
+## 2.10.7
 
 
 ### Bug Fixes
 
 - CI Visibility: Resolves an issue where exceptions other than timeouts and connection errors raised while fetching the list of skippable tests for ITR were not being handled correctly and caused the tracer to crash.
 - CI Visibility: Fixes a bug where `.git` was incorrectly being stripped from repository URLs when extracting service names, resulting in `g`, `i`, or `t` being removed (eg: `test-environment.git` incorrectly becoming `test-environmen`)
-- SSI: Fixes incorrect file permissions on lib-injection images.
-- Code Security: Adds null pointer checks when creating new objects ids.
+- openai: Fixes a bug where `asyncio.TimeoutError`s were not being propagated correctly from canceled OpenAI API requests.
 - profiling: Fixes endpoing profiling for stack v2 when `DD_PROFILING_STACK_V2_ENABLED` is set.
 
 
+---
+
+## 2.9.6
+
+
+### New Features
+
+- SSI: Introduces generic safeguards for automatic instrumentation when using single step install in the form of early exit conditions. Early exit from instrumentation is triggered if a version of software in the environment is not explicitly supported by ddtrace. The Python runtime itself and many Python packages are checked for explicit support on the basis of their version.
```

Comment:
https://github.com/DataDog/dd-trace-py/pull/10579#discussion_r1750816524

Reply:
Wow that's embarrassing, I forgot my own comment 😂 Thanks for catching!